samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -136,7 +140,7 @@ class ButtonAndVirtualGridList extends Component {
 	}
 
 	renderPopup = (rest) => {
-		return <MyVirtualList {...rest} closePopup={this.closePopup} />;
+		return <MyVirtualList {...rest} closePopup={this.closePopup} args={this.props.args} />;
```

Comment:
```suggestion
		return <MyVirtualList {...rest} args={this.props.args} closePopup={this.closePopup} />;
```
You can do like this but if you want to pass some props to `popupComponent`, you can also use `popupProps` prop.

Reply:
I choosed this way following how `closePopup` is passed.